CalculadoraDos/Componets/CustomButton.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  CustomButton.swift
+//  CalculadoraDos
+//
+//  Created by Luis Fernando Sanchez Muñoz on 4/4/24.
+//
+
+import UIKit
+
+final class CustomButton: UIButton {
+    
+    public var digitType: Digitos?
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        custom()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
```

Comment:
En esta función también añadiría custom()
```suggestion
        super.init(coder: coder)
        custom()
```